DLSv2/Utils/Loaders.cs

```diff
@@ -16,39 +13,32 @@ internal class Loaders
     {
         public static Dictionary<Model, DLSModel> ParseVCFs()
         {
-            string path = @"Plugins\DLS\";
-            Dictionary<Model, DLSModel> registeredModels = new Dictionary<Model, DLSModel>();
+            var path = @"Plugins\DLS\";
+            var registeredModels = new Dictionary<Model, DLSModel>();
 
-            // Clear dictionaries, if exists
-            ModeManager.Modes = new Dictionary<Model, Dictionary<string, Mode>>();
-            ControlGroupManager.ControlGroups = new Dictionary<Model, Dictionary<string, ControlGroup>>();
-            AudioControlGroupManager.ControlGroups = new Dictionary<Model, Dictionary<string, AudioControlGroup>>();
-            ControlsManager.ClearInputs();
-            PlayerManager.registeredKeys = false;
-
-            XmlAttributeOverrides attrOverrides = new XmlAttributeOverrides();
+            var attrOverrides = new XmlAttributeOverrides();
             GroupConditions.AddCustomAttributes(attrOverrides);
 
-            XmlSerializer dlsSerializer = new XmlSerializer(typeof(DLSModel), attrOverrides);
+            var dlsSerializer = new XmlSerializer(typeof(DLSModel), attrOverrides);
 
-            foreach (string file in Directory.EnumerateFiles(path, "*.xml"))
+            foreach (var file in Directory.EnumerateFiles(path, "*.xml"))
             {
                 try
                 {
                     DLSModel dlsModel;
-                    using (StreamReader reader = new StreamReader(file))
+                    using (var reader = new StreamReader(file))
                     {
                         dlsModel = (DLSModel)dlsSerializer.Deserialize(reader);
                     }
 
-                    string name = Path.GetFileNameWithoutExtension(file);
+                    var name = Path.GetFileNameWithoutExtension(file);
```

Comment:
Why did you change so many things to use `var`? It's valid, but sometimes it's confusing. 

When you have something like `StreamReader reader = new StreamReader(file)`, it makes sense to use `var` because it's very obviously a `StreamReader` because the constructor is right there. But when you have an arbitrary function whose return type is not necessarily obvious like `Path.GetFileNameWithoutExtension(file)`, you don't necessarily know it'll return a `string`, so I think it's more clear to explicitly define the type as `string`.